doc/migration.md

```diff
@@ -0,0 +1,358 @@
+| `rotatedBy(sf::Angle)` | Get vector rotated by a given angle |
+| `projectedOnto(sf::Vector2)` | Get vector projected onto another vector |
+| `perpendicular()` | Get perpendicular vector |
+| `dot(sf::Vector2)` | Get dot product |
+| `cross(sf::Vector2)` | Get cross product |
+| `cwiseMul(sf::Vector2)` | Perform component-wise multiplication |
+| `cwiseDiv(sf::Vector2)` | Perform component-wise division |
+
+| `sf::Vector3<T>` Function | Description |
+| ------------------------- | ----------- |
+| `length()` |  Get length |
+| `lengthSq()` | Get length squared |
+| `normalized()` | Get vector normalized to unit circle |
+| `dot(sf::Vector2)` | Get dot product |
+| `cross(sf::Vector2)` | Get cross product |
+| `cwiseMul(sf::Vector2)` | Perform component-wise multiplication |
+| `cwiseDiv(sf::Vector2)` | Perform component-wise division |
+
+## Threading Primitives
+
+`sf::Lock`, `sf::Mutex`, `sf::Thread`, `sf::ThreadLocal`, and `sf::ThreadLocalPtr` were removed and replaced with their equivalents from the standard library.
+The standard library provides multiple options for threads, locks, and mutexes among other threading primitives.
+
+| v2                   | v3                                               |
+| -------------------- | ------------------------------------------------ |
+| `sf::Lock `          | `std::lock_guard` or `std::unique_lock`          |
+| `sf::Mutex`          | `std::mutex`                                     |
+| `sf::Thread`         | `std::thread` or `std::jthread` (requires C++20) |
+| `sf::ThreadLocal`    | `thread_local`                                   |
+| `sf::ThreadLocalPtr` | `thread_local`                                   |
+
+## Deprecated APIs
+
+SFML 3 removes all of the deprecated APIs in SFML 2.
+
+| Deprecated API                      | Replacement                        |
+| ----------------------------------- | ---------------------------------- |
+| `sf::Event::MouseWheelEvent`        | `sf::Event::MouseWheelScrolled`    |
+| `sf::RenderWindow::capture`         | See 1                              |
+| `sf::RenderTexture::create`         | `sf::RenderTexture::resize`        |
+| `sf::Shader::setParameter`          | `sf::Shader::setUniform`           |
+| `sf::Text::setColor`                | `sf::Text::setFillColor`           |
+| `sf::Text::getColor`                | `sf::Text::getFillColor`           |
+| `sf::PrimitiveType::LinesStrip`     | `sf::PrimitiveType::LineStrip`     |
+| `sf::PrimitiveType::TrianglesStrip` | `sf::PrimitiveType::TriangleStrip` |
+| `sf::PrimitiveType::TrianglesFan`   | `sf::PrimitiveType::TriangleFan`   |
+| `sf::PrimitiveType::Quads`          | See 2                              |
+
+1. `sf::RenderWindow::capture` can be recreated by using an `sf::Texture` and its `sf::Texture::update(const Window&)` function to copy its contents into an `sf::Image` instead.
+2. `sf::PrimitiveType::Quads` can be replaced by another primitive type.
```

Comment:
A suggestion might still be useful here

Reply:
I wasn't sure what to suggest here since there are a number of alternatives and none of them are simple drop-in replacements.

Reply:
The most common replacement would probably be a `TriangleStrip`. I think it's fine to have the extra sentence, but maybe we can hint at, representing a quad with two triangles facing each other on the long side, or something like that.